google/cloud/storage/client.h

```diff
@@ -2260,7 +2260,7 @@ class Client {
    * @param options a list of optional query parameters and/or request headers.
    *     Valid types for this operation include `UserProject`.
    *
-   * @throw std::runtime_error if there is a permanent failure, or if there were
+   * @returns An error status if there is a permanent failure, or if there were
```

Comment:
There are 6 \@returns in the repo, but 126 \@return.  Is there a difference?  Are we looking for consistency?

Reply:
There is no difference, but we should try to be consistent, fixed.